src/mount/fuse/mount_config.cc

```diff
@@ -155,8 +155,8 @@ void usage(const char *progname) {
 				"of a xor chunk\n"
 "    -o bandwidthoveruse=N       define ratio of allowed bandwidth overuse "
 				"when fetching data (default: %.2f)\n"
-"    -o readcachemaxsize=MB      define the maximum cache memory size "
-				"for reading operations in mebibytes (default: %u)\n"
+"    -o readcachemaxsizepercentage=P  define the maximum cache memory size "
+				"percentage from system for reading operations (default: %u)\n"
```

Comment:
Consider:
Specifies the maximum percentage of system memory for the read cache (default: %u%%)

Reply:
Done